src/Control/Algebra.hs

```diff
@@ -207,16 +217,21 @@ deriving instance Algebra sig m => Algebra sig (Ap m)
 -- @since 1.0.1.0
 deriving instance Algebra sig m => Algebra sig (Alt m)
 
+
```

Comment:
I added some spaces to separate groups of algebras for effects, e.g. grouping the `WriterT` algebras together.